pkg/controller/controller_utils.go

```diff
@@ -703,6 +705,13 @@ func (s ActivePods) Len() int      { return len(s) }
 func (s ActivePods) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 
 func (s ActivePods) Less(i, j int) bool {
+	// 0. Pod with label 'TO_DELETE=true' is smaller
+	_, exists1 := s[i].Labels[POD_DELETE_LABEL]
```

Comment:
and as is mentioned by @kargakis , if the replicas is very large, users will feel painful to label pods manually.